### PR TITLE
Workaround to repair IsLastestVersion for Android.

### LINKED ISSUE #43: Im unable to get GetLatestVersionNumber of the play store. It is always showing empty or null
If this issue is for a new feature, please skip the "Bug" section and fill out the "Feature Request" section at the bottom.

## Bug
Im unable to get GetLatestVersionNumber. It is always showing empty or null
Plugin Version:  2.1.0
Platform (e.g., Android v21):  Andriod 11
Device/Hardware:  

### Steps to reproduce the issue
- 
- 

## Feature Request

Description:

### PR DESCRIPTION
**Pull request by Jerome-Liger**
Quick implementation from Nick Kovalsky workaround :
https://stackoverflow.com/questions/72407251/how-to-get-version-number-of-application-from-play-store-using-xamarin-forms/72643625#72643625

Fixes #43 #42  .  

### Changes proposed in this pull request:  
- Repair plugin for Android Platform.

### CODE REVIEW

**Comment by edsnider** [comment]
Thank you for the PR. 
Please remove the second commit - a test project should not be included in this PR, just the fix. That commit is also breaking the build.   
Thank you.

File: src/Xam.Plugin.LatestVersion.csproj [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <!-- NuGet package info -->
     <PackageId>Xam.Plugin.LatestVersion</PackageId>
     <Title>LatestVersion Plugin for Xamarin and Windows</Title>
-    <PackageVersion>0.1.0</PackageVersion>
+    <PackageVersion>0.1.1</PackageVersion>
```
Comment (edsnider) [inline src/Xam.Plugin.LatestVersion.csproj:15]: Please roll this change back as well. The package version is set by the build pipeline. Thanks!

**Review by edsnider — CHANGES_REQUESTED** [review]
Please undo the package version change in the .csproj (see comment). Also I don't think the changes to the .sln are necessary for the fix, are they? Can those be rolled back as well. Want to keep the commit as minimal/clean as possible. Thank you again for your contribution!

File: src/Xam.Plugin.LatestVersion.csproj (hunk 1 above)
Comment (Jerome-Liger) [reply src/Xam.Plugin.LatestVersion.csproj:15]: Ok.
Yes, the sln changes are not required... but it's automatically modified by my VS2022... I'll try to rollback by modifying the sln manually...

**Comment by edsnider** [comment]
Beta nuget is available here: https://www.nuget.org/packages/Xam.Plugin.LatestVersion/2.1.1-beta.107

**Pull request merged**